docs/managing-dragonfly/operator/dragonfly-configuration.md

```diff
@@ -32,4 +32,4 @@ controller and the dragonfly pods. Below is the table of Dragonfly CRD fields.
 | `topologySpreadConstraints` (since v1.1.1) | \[][TopologySpreadConstraint](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.28/#topologyspreadconstraint-v1-core) | (Optional) Dragonfly pod topologySpreadConstraints |
 | `priorityClassName` (since v1.1.1) | string | (Optional) Dragonfly pod priority class name |
 | `skipFSGroup` (since v1.1.2) | bool | (Optional) Skip Assigning FileSystem Group. Required for platforms such as Openshift that require IDs to not be set, as it injects a fixed randomized ID per namespace into all pods. |
-| `memcachedPort` (since v1.1.2) | int | (Optional) Dragonfly memcached port. Use this instead of `--memcached_port` arg |
\ No newline at end of file
+| `memcachedPort` (since v1.1.2) | int | (Optional) Dragonfly memcached port. Use this instead of `--memcached_port` arg |
```

Comment:
i don't know what's changed in this line. meld shows no diff, and I definitely did not try to modify it.. :shrug:

Reply:
Maybe endline modified or something? Just ignore 😅